pedalboard-web/src/App.vue

```diff
@@ -3,13 +3,11 @@ import Home from './components/Home.vue'
 </script>
 
 <template>
-  <div>
-    <header>
-      <h1>Welcome to PedalBoard!</h1>
-    </header>
-    <div class="content"> 
-      <Home/>
-    </div>
+  <header>
+    <h1>Welcome to PedalBoard!</h1>
+  </header>
```

Comment:
```suggestion
  </header>
  <br>
```

I've noticed that the header and the div "content" are shown snapped! Adding the tag <br> between should separate them